input/fsh/hiv-vocabulary.fsh

```diff
@@ -111,3 +111,59 @@ Description: ""
 * #HIV-ACTUAL-RECENCY-TEST-RESULT "HIV actual recency test result" 
 * #VL-MOST-RECENT-TEST-DATE "VL most recent test date" 
 * #VL-MOST-RECENT-TEST-RESULT "VL most recent test result" 
+
+CodeSystem: CSHIVVLReason
+Id: cs-hiv-vl-reason
+Title: "HIV VL Reason codes"
+Description: "A list of Viral Load test reasons"
+* #BASELINE-VL "Baseline Viral Load"
+* #CHANGE-REGIMEN "Change regimen"
+* #CLINICAL-failure "Clinical failure"
+* #CONFIRMATION-TREATMENT-FAILURE' "Confirmation of treatment failure"
+* #CURRENTLY-BREASTFEEDING-CHILD "Currently breastfeeding child"
+* #CURRENTLY-PREGNANT "Currently pregnant"
+* #PERSISTENT-LOW-LEVEL-VIREMIA "Persistent low-Level viremia"
+* #RECENCY-ASSAY "Recency Assay"
+* #REGIMEN-FAILURE "Regimen failure"
+* #REPEAT "Repeat"
+* #RESULTS-NOT-AVAILABLE "Results not available"
+* #ROUTINE-VIRAL-LOAD "Routine Viral Load"
+
+ValueSet: VSHIVVLReason
+Id: vs-hiv-vl-reason
+Title: "HIV VL Reason codes"
+Description: "Valueset - Viral Load test reasons"
+* include codes from system CSHIVVLReason
+
+CodeSystem: CSARTStatus
```

Comment:
Same above. We cant use snomed or icd or loinc?

Reply:
We have not received the terminology codes, only the UUID's

Reply:
@Lumiere-Lodi please check again, changes committed